app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belong_to :posts, :users
+
+  def update_comment_counter
+    posts.comments_counter += 1
+  end
+end
```

Comment:
- [ ] Comments belong to post as well, please implement it.
- [ ] here you can use class_name: 'User' instead of :users